Options.py

```diff
@@ -41,6 +41,11 @@ def __new__(mcs, name, bases, attrs):
         aliases = {name[6:].lower(): option_id for name, option_id in attrs.items() if
                    name.startswith("alias_")}
 
+        assert (
+            name in {"Option", "VerifyKeys"} or  # base abstract classes don't need default
```

Comment:
Do we support people inheriting directly from `Option`, rather than from `Toggle`, `Range`, `Choice`, etc.?

I think everything that world implementers will use directly already has a `default`, and if we add anything more base in core, we could just add it to this set.

Reply:
I don't know if the option web pages or the yaml generators would be able to handle world implementers using things that we haven't defined with a default.

Reply:
There were two things coming to mind:
1. A prototyped/custom/new option type that that's not in core (yet)
2. A mixin that wants to inherit from Option (to overwrite from_any)

Both are somewhat exotic and both could be worked-around with a simple = None, so I'm fine with it if you think it's fine as is.

Reply:
Yeah, I'm worried about encouraging people to go into complicated territory where they don't need to.

So I think it would be better to keep it more limited like this (with that workaround you mentioned if someone does need to get into it).